src/napari_builtins/builtins.yaml

```
@@ -173,7 +173,7 @@ contributions:
           '*.sti', '*.stk', '*.svs', '*.swf', '*.targa', '*.tf2', '*.tf8',
           '*.tga', '*.tif', '*.tiff', '*.vtk', '*.wap', '*.wbm', '*.wbmp',
           '*.wdp', '*.webm', '*.webp', '*.wmf', '*.wmv', '*.xbm', '*.xpm',
-          '*.zarr', '*.zif'
+          '*.zarr', '*.zarr*', '*.zif'
```

```suggestion
          '*.zarr', '*.zarr/*', '*.zif'
```
I think that we should expect `/` after extension, looking at examples from the description.